queries.sql

```diff
@@ -1,3 +1,11 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+ SELECT * FROM animals WHERE name LIKE '%mon%';
+ SELECT name FROM animals WHERE date_of_birth>= '2016-01-01' AND date_of_birth<= '2019-01-01';
```

Comment:
- [ ] Please make sure that this query selects any birth date between the years `2016` and `2019` which means you have to select until the end of `2019`, meaning `'2019-12-31'` not `'2019-01-01'`.